src/Config/ConfigMerger.php

```diff
@@ -29,10 +29,27 @@ public function merge(Config $config, Config $override = null): Config
             $defaultEnvironment = $override->getDefaultEnvironment();
         }
 
-        return new Config($header, $defaultEnvironment, $environments, $config->getParams());
+        return new Config(new EnvironmentResolver(), $defaultEnvironment, $environments, $config->getParams(), $header);
     }
 
-    private function mergeConfigEnvironments(Config $config, Config $override): array
+    public function mergeImport(Config $config, Config $import = null): Config
```

Comment:
```suggestion
    public function mergeImport(Config $config, ?Config $import = null): Config
```

Reply:
What's the case for that?

Reply:
It's just the correct type, and since php 7.2 is the minimum required version it's possible.